stable/overlays/openstack-base-spaces-overlay.yaml

```diff
@@ -0,0 +1,47 @@
+variables:
+  public-space:        &public-space         public-space
+machines:
```

Comment:
The overlay should not have the machine entries unless you are changing something about the machines.

Reply:
The overlay *must* have matching machine entries, or juju will error like so:

```
ubuntu@icarus-bastion:~/temp-placement/openstack-bundles/stable/openstack-base⟫ juju deploy bundle.yaml --overlay openstack-base-spaces-overlay.yaml
ERROR cannot deploy bundle: the provided bundle has the following errors:
placement "lxd:0" refers to a machine not defined in this bundle
placement "lxd:0" refers to a machine not defined in this bundle
placement "lxd:0" refers to a machine not defined in this bundle
placement "lxd:1" refers to a machine not defined in this bundle
placement "0" refers to a machine not defined in this bundle
placement "lxd:3" refers to a machine not defined in this bundle
placement "lxd:1" refers to a machine not defined in this bundle
placement "lxd:2" refers to a machine not defined in this bundle
placement "lxd:3" refers to a machine not defined in this bundle
placement "1" refers to a machine not defined in this bundle
placement "2" refers to a machine not defined in this bundle
placement "3" refers to a machine not defined in this bundle
placement "lxd:1" refers to a machine not defined in this bundle
placement "lxd:2" refers to a machine not defined in this bundle
placement "lxd:3" refers to a machine not defined in this bundle
placement "lxd:2" refers to a machine not defined in this bundle
placement "1" refers to a machine not defined in this bundle
placement "2" refers to a machine not defined in this bundle
placement "3" refers to a machine not defined in this bundle
```

Take note, commit:
https://github.com/openstack-charmers/openstack-bundles/commit/128f0d3bf972471c405d5e18579b52c1d174c9aa

Reply:
Ok, so that error was because ```machines: {}``` was in the overlay.  Removed the section altogether and that works fine.

Thanks!